docs/BicepInstallation.md

```diff
@@ -0,0 +1,104 @@
+
+### Prerequisites
+To run any of these deployment options, the following items must be set up before execution:
+
+* Contributor and User Access Administrator OR Owner permissions on your Azure subscription 
+
+For local deployments (Options 2 and 3), the following additional steps must be performed:
+* Install the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) module 
+
+* Log into your Azure account and select the subscription you want to deploy the resources in: 
+```PowerShell
+az login 
+```
+
+* If you have more than one subscription, select the subscription you would like to deploy to: 
+```PowerShell
+az account set --subscription <SubscriptionId>
+```
+
+* Clone this repo and navigate to the Bicep deployment folder: 
+```PowerShell
+git clone https://github.com/Microsoft/iomt-fhir
+cd iomt-fhir/deploy/templates/bicep 
+```
+
+### Parameters
+The following parameters are provided by the Bicep template:
+
+|Parameter|Use
+|---|---
+|**Service Name**|Name for the service(s) being deployed. Name will be applied to all relevant services being created.
+|**Resource Location**|The location of the deployed resources.
+|**Resource Identity Resolution Type**|Configures how patient, device, and other FHIR resource identities are resolved from the ingested data stream. The different supported modes are further documented below.
+
+### Resource Identity Resolution Type
+NOTE: All identity look ups are cached once resolved to decrease load on the FHIR server. If you plan on reusing devices with multiple patients it is advised you create a *virtual device* resource that is specific to the patient and the virtual device identifier is what is sent in the message payload. The virtual device can be linked to the actual device resource as a parent.
+
+|Type|Behavior
+|---|---
+|**Create**|System attempts to retrieve the device identifier and associated patient from the FHIR server. If either isn't found a shell resource with just the identity will be created. Requires a patient identifier be mapped in the device content configuration template.
+|**Lookup**|Device identifier from ingested messages is retrieved from the FHIR server. Patient is expected to be linked to the device.
+|**LookupWithEncounter**|Similar to the 'Lookup' setting but allows you to include an encounter identifier with the message to associate with the device/patient. The encounter is looked up during processing and any observations created are linked to the encounter. The association here is assumed to be one encounter per device.
+
+## Deployment 
+### Option 1: Single-click Deploy to Azure via ARM template generated from Bicep Template
+
+### Option 2: Deploy a single Bicep file locally 
+Deploy the [Bicep template](../deploy/templates/bicep/ContainerApp-SingleAzureDeploy.bicep) by running the following command: 
+
+```PowerShell
+az deployment sub create --location <Location> --template-file ContainerApp-SingleAzureDeploy.bicep
```

Comment:
You might want to show an example of how the command should look (e.g., az deployment sub create --location "westus3" --template-file ContainerApp-SingleAzureDeploy.bicep